src/plots/cartesian/layout_defaults.js

```diff
@@ -116,7 +116,7 @@ module.exports = function supplyLayoutDefaults(layoutIn, layoutOut, fullData) {
 
     var bgColor = Color.combine(plot_bgcolor, layoutOut.paper_bgcolor);
 
-    xaList.concat(yaList).forEach(function(axName) {
+    axesList.forEach(function(axName) {
```

Comment:
@mdtusz also from rangesliders, doesn't show up in this diff but master had `axesList.concat(yaList).forEach...` which is obviously redundant.